Subject: Update .gitignore from Debian packaging branch

The Debian packaging git branch contains these updates to the upstream
.gitignore file(s).  This patch is autogenerated, to provide these
updates to users of the official Debian archive view of the package.

[dgit (11.10) update-gitignore]
---
diff --git a/.gitignore b/.gitignore
index 10da9f7..44b20e7 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,5 @@
 *~
 /bin/
+
+/.pc/
+/_build/
